packages/cli/src/platform/apikey/create.ts

```diff
@@ -1,11 +1,13 @@
 import { arg, Command, isError } from '@prisma/internals'
+import { log } from 'console'
```

Comment:
If you could avoid this and use `console.info`, we don't use this at all in the codebase and it's better to be consistent.